Makefile

```diff
@@ -109,16 +109,33 @@ publish-service-invoke: ## Invoke the publish service
 	&& curl -XPOST --fail $$PUBLISH_ENDPOINT
 	@$(PRINT_OK)
 
-decrypt-env:
-	source bin/decrypt-dev-env.sh
-	@$(PRINT_OK)
-
-get-secrets:
-	git submodule update --init --remote
+init-secrets:
+	git clone git@github.com:redbadger/blackbox-secrets.git keyrings -b website-honestly
+	@echo ""
+	@echo "*************************************************************"
+	@echo "* Follow the instructions to get added to the blackbox admins:"
+	@echo "* https://github.com/redbadger/blackbox-secrets/blob/master/README.md#get-access-to-existing-blackbox-secrets"
```

Comment:
this should just go to the the README.md  -the specific heading this goes to get confusing very quickly. it asks for my PGP key, which I've never heard of before. I went to the top of the readme to get more information.

Reply:
I can't add comments to the README, so

https://github.com/redbadger/blackbox-secrets/blob/master/README.md#getting-blackbox

Could this be split out into a series of steps? I think it'd be easier to follow that way.

`as recommended by Blackbox`
recommended where? could you link to official documentation?

I did not see how to set expiration date or comment when I ran `gpg --gen-key` how am I meant to set these?

Reply:
https://github.com/redbadger/blackbox-secrets/blob/master/README.md#get-access-to-existing-blackbox-secrets

`If you do this the first time, you probably need to add yourself as an administrator. Otherwise you won't be able to decrypt the files.`

I'd assume, if someone is on this file, they are doing it for the first steps. How do I add myself as a administrator?

`From the root dir of your project's repository run`
explain the output of this command - i.e. this ... adds your name as an admin. Could you make it clear this is what you are doing?

`If you are not the first admin, ask someone who already has the access to approve your access by reencrypting all files as described in the section Approving someone as an Admin.`
I assume I'm not because you're already in here? Can you add directions on how to find admins?


I think you should write these steps assuming someone is a new contributor to website-honestly.